client/blocks/signup-form/index.jsx

```diff
@@ -6,7 +6,20 @@
 
```

Comment:
Was this file not correctly `prettier`ed? It looks like just whitespace changes.

Reply:
I used to get these issues when I had Atom's Prettier conflicting with Prettier that runs on commit, possibly because we run our own custom fork of Prettier.

Reply:
I got eslint warnings for two non-Map block files, so I committed with `--no-verify`